underlay/src/main/java/bio/terra/tanagra/underlay/indextable/ITInstanceLevelDisplayHints.java

```diff
@@ -24,6 +26,15 @@ public ITInstanceLevelDisplayHints(
     this.entityGroup = entityGroup;
     this.hintedEntity = hintedEntity;
     this.relatedEntity = relatedEntity;
+    this.columnSchemas =
+        ImmutableList.copyOf(
+            Arrays.stream(Column.values()).map(Column::getSchema).collect(Collectors.toList()));
+    this.orderBys =
```

Comment:
Worth sharing this config with the other hints?

Reply:
It is added to both instance and entity level hints already. Did not add it to base class since column schema is set up after call to super constructor in some cases, which is not allowed. I will refactor it and try again in next PR

Reply:
Yeah, I just meant as a constant or something so nobody would forget to update both in the future. Not a big deal.

Reply:
Ah ok. The table columns are defined separately in each class right now, I will refactor